### PR TITLE
Maint: shell scripts polishing

### PR DESCRIPTION
**Pull request by jnpkrn**

### CODE REVIEW

File: test/live_test.sh [hunk 1]
```diff
@@ -677,11 +678,10 @@ run_report() {
 runtest() {
 	local start_ts end_ts
 	local rc booth_status dep_rsc_status
-	local start_time end_time
 	local usrmsg
 	rc=0
 	TEST=$1
-	start_time=`date`
+	start_ts=`date`  # to have the expanded form in the logfile
```
Comment (dmuhamedagic) [inline test/live_test.sh]: This line is superfluous.

File: test/live_test.sh [hunk 2]
```diff
@@ -719,7 +719,7 @@ runtest() {
 		usrmsg="test FAIL: $rc"
 		;;
 	esac
-	end_time=`date`
+	end_ts=`date`  # to have the expanded form in the logfile
```
Comment (dmuhamedagic) [inline test/live_test.sh]: This one too.

File: test/live_test.sh [hunk 3]
```diff
@@ -620,7 +621,7 @@ check_booth_consistency() {
 `if [ $rc -ge 4 ]; then
 	echo "booth list consistency failed (more than one leader!):"
 elif [ $rc -ge 2 ]; then
-	echo "booth list consistency failed (some boots not uptodate):"
+	echo "booth list consistency failed (some booths not up-to-date):"
```
Comment (dmuhamedagic) [inline test/live_test.sh]: Must say that I occasionally enjoyed seeing boots ;-)

**Comment by jnpkrn** [comment]
Yes, it's superfluous per se, but I though the intention was for those lines
to be captured thanks `set -x` ... in that light, what was disputably superfluous
was to occupy brand new variables without using them later on, which is
something ShellCheck screams about. That's what the comments mean to say.

**Comment by jnpkrn** [comment]
@dmuhamedagic Does the above explanation suffice to proceed with this PR?

**Comment by dmuhamedagic** [comment]
See what you mean. Those variables were introduced for whatever reason other than logging, then apparently forgotten, at least I cannot recall otherwise.

**Comment by dmuhamedagic** [comment]
Not that it really matters, but I guess using ':' would be a somewhat more apparent way.

**Comment by jnpkrn** [comment]
On 13/08/18 22:42 -0700, Dejan Muhamedagic wrote:
> Not that it really matters, but I guess using ':' would be
> a somewhat more apparent way.

True, that would be a good use of the dummy command.

And thanks for merging.

-- 
Jan (Poki)

**Pull request merged**